theme/src/scss/_copilot.scss

```diff
@@ -13,8 +13,8 @@
     // Popovers on site use z-index: 20
     z-index: 10;
 
-    // Default use up the full height of the viewport
-    top: 0;
+    // Default, make space for the navbar
+    top: calc(38px + 8px + 8px + 54px);
```

Comment:
Unless we find the breakdown of this calculation useful, should this just be written out explicitly instead as it is not clear what 38, 8, 8, 54 represents? i.e. `top: 108px`, since that is what this would resolve to.

Reply:
Good one.